mbzirc_ign/CMakeLists.txt

```diff
@@ -81,14 +81,25 @@ ament_python_install_package(
 #============================================================================
 # Resources
 install(DIRECTORY
+  config
   worlds
   models
   launch
   ${CMAKE_CURRENT_BINARY_DIR}/hooks
   DESTINATION share/${PROJECT_NAME})
 
 if(BUILD_TESTING)
+  find_package(python_cmake_module REQUIRED)
+  find_package(PythonExtra MODULE REQUIRED)
+
   find_package(ament_cmake_gtest REQUIRED)
+  find_package(ament_cmake_pytest REQUIRED)
+
+  find_package(ament_cmake_flake8 REQUIRED)
+  ament_flake8()
+
+  find_package(ament_cmake_pycodestyle REQUIRED)
+  ament_pycodestyle()
```

Comment:
hmm maybe we should have instructions in `README.md` to have a way to avoid installing packages for tests when running `rosdep` in step 3 (with `t`?).

Reply:
Yes, this is probably more convenient for us.  I would even be okay not turning them on, but I think it catches a bare minimum of potential python issues.

Reply:
yeah we can keep it on, and just offer instructions for users to turn it off.